modules/apps/object/object-service/src/main/java/com/liferay/object/internal/petra/sql/dsl/DynamicObjectDefinitionTable.java

```diff
@@ -73,7 +73,8 @@ public DynamicObjectDefinitionTable(
 			if (objectField.compareBusinessType(
 					ObjectFieldConstants.BUSINESS_TYPE_AGGREGATION) ||
 				objectField.compareBusinessType(
-					ObjectFieldConstants.BUSINESS_TYPE_FORMULA)) {
+					ObjectFieldConstants.BUSINESS_TYPE_FORMULA) ||
+				objectField.isLocalized()) {
```

Comment:
Create a new method inside `ObjectField` and call it `isStoredInDynamicTables`

Reply:
@gabrielwas I'm not sure if we should create a method that will be:

````java
public boolean isStoredInDynamicTables() {
  return isLocalized();
}
````

what do you think?